Mathlib/Topology/Separation/CompletelyRegular.lean

```diff
@@ -103,6 +103,33 @@ lemma Topology.IsInducing.completelyRegularSpace
 instance {p : X → Prop} [CompletelyRegularSpace X] : CompletelyRegularSpace (Subtype p) :=
   Topology.IsInducing.subtypeVal.completelyRegularSpace
 
+lemma CompletelyRegularSpace.isInducing_stoneCechUnit [CompletelyRegularSpace X] :
```

Comment:
I think you can get away without the namespace here, especially in light of the `↔` below.
```suggestion
lemma isInducing_stoneCechUnit [CompletelyRegularSpace X] :
```